### PR TITLE
KFSTI-1371 Add logging to PropertyLoadingFactoryBean

### PR DESCRIPTION
**Pull request by kkronenb**
The sysout is a temp thing so I don't have to fight log4j too much yet.

### CODE REVIEW

File: src/main/java/org/kuali/kfs/sys/context/PropertyLoadingFactoryBean.java [hunk 1]
```diff
@@ -32,6 +25,14 @@
 import org.springframework.beans.factory.FactoryBean;
 import org.springframework.core.io.DefaultResourceLoader;
 
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.Arrays;
```
Comment (smith750) [inline src/main/java/org/kuali/kfs/sys/context/PropertyLoadingFactoryBean.java]: Did you need to import all of these?

File: src/main/java/org/kuali/kfs/sys/context/PropertyLoadingFactoryBean.java [hunk 2]
```diff
@@ -65,7 +69,14 @@ public Properties getObject() {
         else {
             props.put(KSB_REMOTING_URL_PROPERTY_NAME, new StringBuilder("http://").append(System.getProperty(HTTP_URL_PROPERTY_NAME)).append("/kfs-").append(props.getProperty(KFSConstants.ENVIRONMENT_KEY)).append(REMOTING_URL_SUFFIX).toString());
         }
-        
+        if (LOG.isDebugEnabled()) {
+            for (Iterator itr = props.keySet().iterator(); itr.hasNext(); ) {
+                String key = (String) itr.next();
```
Comment (smith750) [inline src/main/java/org/kuali/kfs/sys/context/PropertyLoadingFactoryBean.java]: The iterator couldn't be genericized?  And we needed an iterator here?  yuck...

File: src/main/java/org/kuali/kfs/sys/context/PropertyLoadingFactoryBean.java [hunk 3]
```diff
@@ -97,6 +108,7 @@ private static void loadProperties( Properties props, String propertyFileName) {
             }
         }
         catch (IOException e) {
+            e.printStackTrace();
```
Comment (smith750) [inline src/main/java/org/kuali/kfs/sys/context/PropertyLoadingFactoryBean.java]: We're not logging the stack trace, just dumping to System.err?

File: src/main/java/org/kuali/kfs/sys/context/PropertyLoadingFactoryBean.java (hunk 1 above)
Comment (kkronenb) [inline src/main/java/org/kuali/kfs/sys/context/PropertyLoadingFactoryBean.java]: intellij just moved them. They were there before

**Pull request merged**